apis/core/v1beta1/featureflag_webhook_test.go

```diff
@@ -0,0 +1,167 @@
+package v1beta1
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_validateFeatureFlagTargeting(t *testing.T) {
+	tests := []struct {
+		name    string
+		in      Flags
+		wantErr bool
+	}{
+		{
+			name: "happy path",
```

Comment:
Potentially more unit tests can be added from [here](https://github.com/open-feature/flagd-schemas/blob/main/json/test/negative/invalid-flagd-props.json), but in my eyes, more tests are testing the schema, not the validation webhook itself

Reply:
Agreed. I think as long as you test 1 failure and 1 success that's sufficient for OFO. This is more than enough.